components/application-operator/charts/gateway/templates/deployment.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.global.deployGatewayOncePerNamespace }}
```

Comment:
I think in the case of disabling Gateway per namespace we can just not create the Service Instance Controller.